R/parser.R

```diff
@@ -819,7 +819,14 @@ parse_mbox <- function(perceval_path,mbox_path){
 #' @family parsers
```

Comment:
build the documentation (command shift d on mac) and the project again (command shift b on mac) before commits so unit checks pass

Reply:
was this done?